src/postgres/getPostgresTablesQueryRows.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ClientConfig } from "pg";
+import { runPostgresQuery } from "./runPostgresQuery";
+
+export interface IPostgresTable {
+    schemaName: string;
+    name: string;
+    columnNames: string[];
+}
+
+function getTableInfo(): string {
+    return `select schemaname, tablename, array_agg (columnname) as columnarray
+            from pg_catalog.pg_tables
+            left join (select column_name::text as columnname, table_name, table_schema from information_schema.columns) columns on table_name = tablename and table_schema = schemaname
```

Comment:
Could you explain what `::text` does in `select column_name::text ...`?

Reply:
The `column_name::text` parses column_name to text data type so that it can be aggregated later, otherwise the original data type of column_name is information_schema.sql_identifier.